packages/hooks/use-infinite-scroll/tests/index.test.tsx

```diff
@@ -0,0 +1,283 @@
+          </div>
+          <button onClick={() => resetRef.current()} data-testid="reset">
+            Reset
+          </button>
+        </>
+      )
+    }
+
+    const { getByTestId, user } = render(<MyComponent />)
+    vi.spyOn(window, "scrollTo").mockImplementation(() => {
+      document.body.scrollTop = 100
+    })
+    const reset = getByTestId("reset")
+    await user.click(reset)
+
+    await waitFor(() => {
+      expect(document.body.scrollTop).toBe(100)
+    })
+  })
+
+  test("should be called onReset reverse scrooll to root", async () => {
+    const MyComponent = () => {
+      const mockOnLoad = vi.fn().mockImplementation(async ({ finish }) => {
+        finish()
+      })
+      const rootRef = useRef<HTMLDivElement>(null)
+      const resetRef = useRef<() => void>(() => {})
+      const { ref, isFinish } = useInfiniteScroll({
+        onLoad: mockOnLoad,
+        rootRef,
+        resetRef,
+        isReverse: true,
+      })
+
+      return (
+        <>
+          <div ref={rootRef} style={{ overflow: "auto" }}>
+            {!isFinish ? <div ref={ref} data-testid="trigger" /> : null}
+          </div>
+          <button onClick={() => resetRef.current()} data-testid="reset">
+            Reset
+          </button>
+        </>
+      )
+    }
+
+    const { container, getByTestId, user } = render(<MyComponent />)
+    const root = container.firstChild as HTMLDivElement
+    vi.spyOn(root, "scrollTo").mockImplementation(() => {
+      root.scrollTop = 100
+    })
+    const reset = getByTestId("reset")
+    await user.click(reset)
+
+    await waitFor(() => {
+      expect(root.scrollTop).toBe(100)
+    })
+  })
+
+  test("should be called onReset horizontal reverse scrooll to root", async () => {
+    const MyComponent = () => {
+      const mockOnLoad = vi.fn().mockImplementation(async ({ finish }) => {
+        finish()
+      })
+      const rootRef = useRef<HTMLDivElement>(null)
+      const resetRef = useRef<() => void>(() => {})
+      const { ref, isFinish } = useInfiniteScroll({
+        onLoad: mockOnLoad,
+        rootRef,
+        resetRef,
+        isReverse: true,
+        orientation: "horizontal",
+      })
+
+      return (
+        <>
+          <div ref={rootRef} style={{ overflowX: "auto" }}>
+            {!isFinish ? <div ref={ref} data-testid="trigger" /> : null}
+          </div>
+          <button onClick={() => resetRef.current()} data-testid="reset">
+            Reset
+          </button>
+        </>
+      )
+    }
+
+    const { container, getByTestId, user } = render(<MyComponent />)
+    const root = container.firstChild as HTMLDivElement
+    vi.spyOn(root, "scrollTo").mockImplementation(() => {
+      root.scrollLeft = 100
+    })
+    const reset = getByTestId("reset")
+    await user.click(reset)
+
+    await waitFor(() => {
+      expect(root.scrollLeft).toBe(100)
+    })
+  })
+
+  test("should be called onReset horizontal reverse scrooll to body", async () => {
```

Comment:
It seems not to be the reverse.
```suggestion
  test("should be called onReset horizontal scrooll to body", async () => {
```

Reply:
@Neru-Neru 

Thx! I fixed it.
612cbd135690b3b8271ae90d2216ed611eb7d8af